build_node/build_node_builder.py

```diff
@@ -198,7 +211,21 @@ def __report_excluded_task(self, task, artifacts):
             'status': 'excluded',
             'artifacts': [artifact.dict() for artifact in artifacts]
         }
-        self.__call_master('build_done', **kwargs)
+        traceback_str = ''
+        try:
+            self.__call_master('build_done', **kwargs)
+        except requests.exceptions.Timeout:
+            traceback_str = traceback.format_exc()
+            self.__retry_counter -= 5
```

Comment:
Do we really need to decrement retry counter here? This will lead to situation that we will exceed retries too quick.